src/test/resources/uk/gov/ons/ctp/journeys/load/sample/censusLoadSample.feature

```diff
@@ -11,7 +11,9 @@
 # Feature Tags: @loadSample
 #               @censusLoad
 #
-@loadSample @censusLoad
+
+# Test fails until defect CTPA-1691 is resolved
```

Comment:
What's CTPA-1691? Are we commenting out the tags so the test doesn't run? If so we might be better off just using the predefined skip tag https://jenisys.github.io/behave.example/tutorials/tutorial11.html

Reply:
CTPA-1691 is a task on Jira from titchfield. I believe we are commenting the tests out to be consistent with what has already been done in the repo. There is a card on the board for going through the commented out tests in the repo at a later date if needed.

Reply:
The "skip" annotation does not work in cucumber the same way as behave. In cucumber we use _"~<tag_name>"_ or _"not <tag_name>"_ to skip tags. But this will require changing Jenkins jobs as well. For now, I have added an additional comment to clarify that we are skipping all tests by commenting tags. We will introduce new tags to skip scenarios when we re-visit these features to fix the commented scenarios.